pkg/util/util.go

```diff
@@ -129,10 +131,7 @@ func AddOrUpdateBinding(existingBindings []*cloudresourcemanager.Binding, requir
 
 // roleBindingMap returns a map of requiredBindings role bindings for the added members
 func rolebindingMap(roles []string, member string, memberType IamMemberType) map[string]cloudresourcemanager.Binding {
-	prefix := "serviceAccount:"
-	if memberType == GoogleGroup {
```

Comment:
We may still need this check depending on what membertype is returned.

Reply:
We will leave this to the original logic and have a card in to refactor in the future.